### DISCUSSION TITLE
Bug Report: Expected } error

### DISCUSSION THREAD

**Discussion by ThisIsVortexx** [KadeArchive/Kade-Engine#435] [Q&A]
In the playstate.hx file i have this typed in

    else if (SONG.song.toLowerCase() == 'Snowflake'){
						curStage = 'winterstage';
						
						var bg:FlxSprite = new FlxSprite( -3557, -1974); 
						bg.loadGraphic(Paths.image("winter/bg"));
						add(bg);
						bg.active = false;
						bg.updateHitbox();
						bg.antialiasing = true
						bg.scrollFactorset(0.9, 0.9);
						
					}

But it doesnt register and gives me and "Expected }" error
Can anyone help?

**Comment by poco0317** [comment]
missing ;

**Comment by ThisIsVortexx** [comment]
> missing ;

Where??
@poco0317

**Comment by isakube** [comment]
bg.antialiasing = true;

**Comment by ThisIsVortexx** [comment]
> bg.antialiasing = true;

Yeah i tried that too i added all the semi colons but it still shows me the same thing

**Comment by isakube** [comment]
count the number of bracket pairs ig?? maybe you forgot to add a closing bracket to something
